crates/edr_solidity_tests/tests/it/fork.rs

```diff
@@ -11,7 +11,7 @@ async fn test_cheats_fork_revert() {
     let filter = SolidityTestFilter::new(
         "testNonExistingContractRevert",
         ".*",
-        &format!(".*cheats{RE_PATH_SEPARATOR}Fork"),
+        &format!(".*cheats{RE_PATH_SEPARATOR}Fork2"),
```

Comment:
Is this a unique identifier that needs to be upgraded any time the filter changes?

Reply:
The intention here is to match a single test suite, but there are actually two test suites with the `cheats/Fork` prefix: `cheats/Fork.t.sol` and `cheats/Fork2.t.sol`. Here we want to only match the latter, and this was achieved previously by also filtering the test methods. But with the change in semantics in this PR, if the path/suite name matches, but the test name filter doesn't match, we will return an empty suite result. So in order to only get one test suite result here, I had to make the path filter more specific to only match `Fork2.t.sol`.

Reply:
Gotcha! Thanks for explaining 🙂